versions.yaml

```diff
@@ -18,18 +18,18 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 versions:
-  - dir: '2/2.150'
```

Comment:
K8s app uses this version.

Reply:
https://github.com/GoogleCloudPlatform/click-to-deploy/blob/ed00c99c33ee6d63cfdbcf91e56577775e7da93b/k8s/jenkins/Makefile#L9

Reply:
Right, I restored 2.150 version and removed 2.138 as it is no longer used by app